app/src/test/java/org/simple/clinic/teleconsultlog/prescription/TeleconsultPrescriptionUpdateTest.kt

```diff
@@ -52,14 +52,34 @@ class TeleconsultPrescriptionUpdateTest {
         .whenEvent(DataForNextClickLoaded(
             medicalInstructions = medicalInstructions,
             medicalRegistrationId = medicalRegistrationId,
-            hasSignatureBitmap = false
+            hasSignatureBitmap = false,
+            hasMedicines = true
         ))
         .then(assertThatNext(
             hasNoModel(),
             hasEffects(ShowSignatureRequiredError)
         ))
   }
 
+  @Test
+  fun `when there are no medicines added, then show medicines required error`() {
+    val medicalInstructions = "This is a medical instructions"
+    val medicalRegistrationId = "ABC12345"
+
+    updateSpec
+        .given(model)
+        .whenEvent(DataForNextClickLoaded(
```

Comment:
Can we add `DataForNextClickLoaded` event as a local property for better readability?

Reply:
we can, but we are not reusing it anywhere and IMO the implementation is not large enough to extract to an variable 🤔

Reply:
Sure, no problem. Had readability in mind., but this works as well.